diff --git a/src/main/java/ipsis/woot/command/CommandWoot.java b/src/main/java/ipsis/woot/command/CommandWoot.java
index f6c1e1a..300a995 100644
--- a/src/main/java/ipsis/woot/command/CommandWoot.java
+++ b/src/main/java/ipsis/woot/command/CommandWoot.java
@@ -54,7 +54,7 @@
     public void execute(MinecraftServer server, ICommandSender sender, String[] args) throws CommandException {
 
         if (args.length == 0)
-            throw new WrongUsageException("commands.Woot:woot.usage");
+            throw new WrongUsageException("commands.woot:woot.usage");
         else if ("dev".equals(args[0]))
             dev(server, sender,args);
         else if ("flush".equals(args[0]))
@@ -76,13 +76,13 @@
         else if ("cost".equals(args[0]))
             Woot.mobRegistry.cmdDumpCosts(sender);
         else
-            throw new WrongUsageException("commands.Woot:woot.usage");
+            throw new WrongUsageException("commands.woot:woot.usage");
     }
 
     private void dev(MinecraftServer server, ICommandSender sender, String[] args) throws CommandException {
 
         if (args.length != 2)
-            throw new WrongUsageException("commands.Woot:woot.usage");
+            throw new WrongUsageException("commands.woot:woot.usage");
 
         boolean b = false;
         String v = args[1];
@@ -91,16 +91,16 @@
         else if (v.equalsIgnoreCase("false"))
             b = false;
         else
-            throw new WrongUsageException("commands.Woot:woot.usage");
+            throw new WrongUsageException("commands.woot:woot.usage");
 
         Woot.devMode = b;
-        sender.sendMessage(new TextComponentTranslation("commands.Woot:woot.dev.summary", v.toLowerCase()));
+        sender.sendMessage(new TextComponentTranslation("commands.woot:woot.dev.summary", v.toLowerCase()));
     }
 
     private void flush(ICommandSender sender, String[] args) throws CommandException {
 
         if (args.length < 2)
-            throw new WrongUsageException("commands.Woot:woot.usage");
+            throw new WrongUsageException("commands.woot:woot.usage");
 
         String mobName = args[1];
 
@@ -117,7 +117,7 @@
     private void give(MinecraftServer server, ICommandSender sender, String[] args) throws CommandException {
 
         if (args.length != 4)
-            throw new WrongUsageException("commands.Woot:woot.usage");
+            throw new WrongUsageException("commands.woot:woot.usage");
 
         EntityPlayer entityplayer = getPlayer(server, sender, args[1]);
         String wootName = args[2];
@@ -166,7 +166,7 @@
     private void loot(MinecraftServer server, ICommandSender sender, String[] args) throws CommandException {
 
         if (args.length < 2)
-            throw new WrongUsageException("commands.Woot:woot.usage");
+            throw new WrongUsageException("commands.woot:woot.usage");
 
         String mobName = args[1];
         boolean detail = false;
diff --git a/src/main/java/ipsis/woot/manager/loot/LootTableManager.java b/src/main/java/ipsis/woot/manager/loot/LootTableManager.java
index c8b063e..ffce750 100644
--- a/src/main/java/ipsis/woot/manager/loot/LootTableManager.java
+++ b/src/main/java/ipsis/woot/manager/loot/LootTableManager.java
@@ -161,7 +161,7 @@
         for (String mobName : lootMap.keySet())
             sb.append(mobName).append(" ");
 
-        sender.sendMessage(new TextComponentTranslation("commands.Woot:woot.mobs.summary", sb));
+        sender.sendMessage(new TextComponentTranslation("commands.woot:woot.mobs.summary", sb));
     }
 
     public void dumpDrops(ICommandSender sender, String wootName, boolean detail) {
@@ -170,7 +170,7 @@
         if (e != null) {
             for (EnumEnchantKey key : EnumEnchantKey.values()) {
                 String s = e.getDrops(key, detail);
-                sender.sendMessage(new TextComponentTranslation("commands.Woot:woot.loot.summary", wootName, key.getDisplayName(), s));
+                sender.sendMessage(new TextComponentTranslation("commands.woot:woot.loot.summary", wootName, key.getDisplayName(), s));
             }
         }
     }
@@ -180,7 +180,7 @@
         LootTable e = lootMap.get(wootName);
         if (e != null) {
             e.flush(key);
-            sender.sendMessage(new TextComponentTranslation("commands.Woot:woot.flush.summary",
+            sender.sendMessage(new TextComponentTranslation("commands.woot:woot.flush.summary",
                     wootName, key.getDisplayName()));
         }
     }
@@ -194,7 +194,7 @@
             table.flush(EnumEnchantKey.LOOTING_III);
         }
 
-        sender.sendMessage(new TextComponentTranslation("commands.Woot:woot.flush.all.summary"));
+        sender.sendMessage(new TextComponentTranslation("commands.woot:woot.flush.all.summary"));
     }
 
     public void dumpBlacklist(ICommandSender sender) {
@@ -203,7 +203,7 @@
         for (ItemStack itemStack : blacklist)
             sb.append(String.format("%s ", itemStack.getDisplayName()));
 
-        sender.sendMessage(new TextComponentTranslation("commands.Woot:woot.blacklist.summary", sb.toString()));
+        sender.sendMessage(new TextComponentTranslation("commands.woot:woot.blacklist.summary", sb.toString()));
     }
 
     public void dumpStatus(ICommandSender sender) {
@@ -220,7 +220,7 @@
                 sb.append(" " + key.ordinal() + ":" + samples + "/" + Settings.sampleSize + " ");
             }
             sender.sendMessage(new TextComponentTranslation(
-                    "commands.Woot:woot.status.summary",
+                    "commands.woot:woot.status.summary",
                     mob, sb.toString()));
         }
     }
